natlas-server/app/static/natlas.scss

```diff
@@ -447,3 +585,70 @@ button.check-update {
     transform: translate(-50%, -50%);
     transition: 0.3s;
 }
+
+.landing-content {
+    width:75%;
+    margin:0 auto;
+    text-align: center;
+}
+
+.landing-label {
+    font-size: 1.5em;
+    margin-bottom:0.5em;
+    margin-top: 0.5em;
+    display: block;
+}
+
+.landing-input {
+    border: none;
+    line-height:3em;
+    border-radius:0.5em;
+    width:100%;
+    padding-left:0.5em;
+    padding-right:0.5em;
+    pointer-events:all;
+}
+
+.landing-button {
+    width: 100%;
+}
+
+.capture-pointer {
+    pointer-events: all;
+}
+
+.login-links {
+    justify-content: space-between;
+    width: 100%;
+}
+
+#particles-js{
+    position:absolute;
+    width: 100%;
+    height: 100%;
+}
+
+#searchlayer {
+    z-index:100;
+    width: 700px;
+}
+
+.aligner {
+    display: flex;
+    align-items: start;
+    justify-content: center;
+    height: 100%;
+    pointer-events: none;
+}
+
+.st0{fill:url(#SVGID_1_);}
+.st1{fill:url(#SVGID_2_);}
+.st2{fill:url(#SVGID_3_);}
+.st3{fill:url(#SVGID_4_);}
+.st4{fill:url(#SVGID_5_);}
+.st5{fill:url(#SVGID_6_);}
```

Comment:
wtf?

Reply:
This is svg gradient stuff. I can embed this as a `<style>` tag into the svg, which is where it was originally.